src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/DocumentsOperation.java

```diff
@@ -69,11 +69,11 @@ public DocumentsOperation(final SecurityUtils securityUtils,
             return Arrays.asList(documents);
         } catch (Exception e) {
             LOG.error(String.format(
-                "Cannot get documents for the Jurisdiction:%s, Case Type Id:%s, Case Reference:%s",
-                jurisdictionId, caseTypeId, caseReference), e);
+                "Cannot get documents for the Jurisdiction:%s, Case Type Id:%s, Case Reference:%s, because of %s",
+                jurisdictionId, caseTypeId, caseReference, e.getMessage()), e);
             throw new ServiceException(String.format("Cannot get documents for the Jurisdiction:%s, Case Type Id:%s, "
                     + "Case Reference:%s, because of %s",
-                jurisdictionId, caseTypeId, caseReference, e));
+                jurisdictionId, caseTypeId, caseReference, e.getMessage()), e);
```

Comment:
Ideally, even the message inside Exception should not be communicated to the client.

Reply:
Fixed